commons/commons-util/src/main/kotlin/ebi/ac/uk/util/file/ExcelReader.kt

```diff
@@ -1,13 +1,21 @@
 package ebi.ac.uk.util.file
 
+import com.monitorjbl.xlsx.StreamingReader
 import ebi.ac.uk.dsl.Tsv
 import ebi.ac.uk.dsl.TsvLine
-import org.apache.poi.xssf.usermodel.XSSFWorkbook
 import java.io.File
 
+const val BUFFER_SIZE = 4096
+const val ROW_CACHE_SIZE = 1000
+
 class ExcelReader {
     fun readContentAsTsv(file: File): String {
-        val sheet = XSSFWorkbook(file.inputStream()).getSheetAt(0)
+        val sheet = StreamingReader.builder()
```

Comment:
don't we need to close the buffered reader?

Reply:
Good catch. Done.